presto-main/src/main/java/com/facebook/presto/execution/QueryExecution.java

```diff
@@ -45,7 +50,7 @@ Duration waitForStateChange(QueryState currentState, Duration maxWait)
 
     Session getSession();
 
-    void start(Optional<String> resourceGroupName);
```

Comment:
Why did you change this to be a setter, instead of passing it in the start() method?